### DISCUSSION TITLE
Slate Rich Text Editor from Payload UI in my frontend React app?

### DISCUSSION THREAD

**Discussion by hai-qiu-bpc** [payloadcms/payload#3185] [Q&A]
I have a frontend end built with React and use Rest Api calls to communicate with Payload.
I want to implement a cms editor in my fronend. I want to have a rich text editor in my frontend, and send the saved content to my payload/DB.

Is it possible to have the same rich text editor displayed in payload cms in my frontend as well. e.g. as react component or something like that. But the content would still be saved the same way as in the original payload ui.

How can I approach that?

**Comment by denolfe** [comment]
This is not possible currently. We have some upcoming work to make [Payload's components to be reusable](https://github.com/payloadcms/payload/discussions/293). However, I would not expect the rich text editor to be part of this since it is so coupled to Payload.
